share/wake/lib/system/job.wake

```diff
@@ -404,7 +404,7 @@ export def preloadRunner =
   makeJSONRunnerPlan preload score
   | makeJSONRunner
 
-def rOK = 0
+def _rOK = 0 # unused; remove _ if you use it
 def wOK = 1
 def xOK = 2
 def access file mode = prim "access"
```

Comment:
This should be export/global anyway I think, we have 2 places where it's copy/pasted already

Reply:
You're probably right. I'll export them.